package/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { createResolver } from "./utils/create-resolver";
 export { watchIntegration } from "./utils/watch-integration";
+export { addVirtualImport } from "./utils/add-virtual-import";
```

Comment:
Running `pnpm lint:fix` should have moved this import to the top 🤔

Reply:
I've moved it manually to the top for now